src/__experimental__/components/textbox/textbox.stories.js

```diff
@@ -40,6 +58,33 @@ storiesOf('Experimental/Textbox', module)
         { ...getCommonTextboxStoryProps() }
       />
     ]);
+  }, {
+    info: {
+      text: info,
+      propTables: [OriginalTextbox],
+      propTablesExclude: [State, Textbox],
+      notes: { markdown: notes }
```

Comment:
I think textbox needs to handle special characters
`knobs: {
   escapeHTML: false
}`

Reply:
added